src/codegen/riscv64/constants-riscv64.h

```diff
@@ -978,6 +978,44 @@ class InstructionGetters : public T {
     return imm6 << 26 >> 26;
   }
 
+  inline int RvcImm6LwspValue() const {
```

Comment:
If it extracts 8-bit immediate value, better be called `RvcImm6LwspValue`. Otherwise, if only 6-bit value is extracted, then the variable should be called `imm6` (instead of `imm8`).

Reply:
`RvcImm8LwspValue` would cause confusing because 8 here doesn't mean the number of bits.

As for `RvcImm6ScaledLwspValue`, I wonder whether it is necessary to show the detail `Scaled` in function name. As an abstraction, I think this detail should be hidden. Besides, these methods' return value type are the same, int. The caller would not realize the difference between these functions and 32-bit immediate functions.

I prefer the current name bacause

1. consistence. They are consistency w/ 32-bit immediate functions.

2. minimal distiguishment. This would ignore unnecessary information for callers.

Reply:
ok